meta-flex-distro/conf/distro/flex-os.conf

```diff
@@ -205,8 +205,8 @@ SYSTEMD_DEFAULT_TARGET ?= '${@bb.utils.contains_any("IMAGE_FEATURES", ["x11-base
 # Prefer docker-moby to docker-ce by default
 PREFERRED_PROVIDER_virtual/docker = "docker-moby"
 
-# Add extra-space to rootfs (default: 204800KB) for virtualization
-IMAGE_ROOTFS_EXTRA_SPACE:append = "${@bb.utils.contains('DISTRO_FEATURES', 'virtualization', ' + 307200', '', d)}"
+# Add extra-space to rootfs (default: 716800KB) for virtualization in absence of flex-ewaol-baremetal
+IMAGE_ROOTFS_EXTRA_SPACE:append:feature-virtualization = "${@bb.utils.contains('DISTRO_FEATURES', 'flex-ewaol-baremetal', '', ' + 716800', d)}"
```

Comment:
In absence of flex-ewaol-baremetal? Isn't it the opposite?

Secondly the "feature-virtualization" override won't be necessary if you're going to check on flex-ewaol-baremetal. 

P.S. have you confirmed that this works? flex-ewaol-baremetal doesn't land in the DISTRO_FEATURES until after the distro conf has been parsed, which this file is a part of.